src/sentry/static/sentry/app/views/performance/landing/vitalsCards.tsx

```diff
@@ -104,6 +117,7 @@ export function FrontendCards(props: VitalsCardsProps) {
                     tooltip={vitalDescription[vital] ?? ''}
                     value={isLoading ? '\u2014' : value}
                     chart={chart}
+                    minHeight={150}
```

Comment:
Is this for the sparkline scaling? You can just make that part of the css, not sure it needs to be a prop.

Reply:
This is for making sure the frontend cards don't grow when the colour bars are finished loading. Though I think you're right that I should be able to fix the colour bar height so this might not need to be a prop. Will address in a follow up.